### ISSUE TITLE
 batch size>1

### ISSUE THREAD

**Issue by dongbo811** [uta-smile/ASSUDA#2]
Hi I have a question. If I set the batch size greater than 1, and the code will report an error. However, the 2N mentioned in paper description does not seem to correspond.

**Comment by viyjy** [comment]
Hi, we have only tried batch size=1. What do you mean by "the 2N mentioned in paper description does not seem to correspond"?

**Comment by dongbo811** [comment]
thx a lot for your answer.
On this page, if we set the minibatch to N, we can get 2N samples.
But, we found that it is not possible in the experiment.


















At 2021-12-13 02:25:56, "viyjy" ***@***.***> wrote:

Hi, we have only tried batch size=1. What do you mean by "the 2N mentioned in paper description does not seem to correspond"?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by viyjy** [comment]
You are right, this code can only be applied to batch size=1, since we found that most of existing UDA methods in semantic segmentation use batch size=1. I can provide a new one which can handle batch size > 1. Before that, can you use batch size=1? Thanks.
